crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -34,6 +34,20 @@ def check
       }
     end
 
+    def target_platform(options = {})
+      # FIXME: get rid of platform_exception when SES upgraded to SP2
```

Comment:
Actually, I think it's fine to keep it. The current code is generic, so it will keep working in that case.